### PR TITLE
chore: set dir to absolute path

### PR DESCRIPTION
**Pull request by Decodetalkers**
I want to set this variable to absolute path, 

because when debug with fakegreetd "river -c bin"

the current dir of `bin` is not the current dir of fakegreetd, the current dir  is `/`, so bin cannot find `./greet.sock`.

### CODE REVIEW

**Comment by Decodetalkers** [comment]
and if the log can switch to tracing? because with out color, I maybe cannot know the log is from fakegreetd

File: fakegreet/src/main.rs [hunk 1]
```diff
@@ -151,12 +151,21 @@ async fn client_handler(ctx: &Context, mut s: UnixStream) -> Result<(), Error> {
...
-    std::env::set_var("GREETD_SOCK", path);
-
-    let _ = std::fs::remove_file(path);
+    let mut path = env::current_dir()?;
+    let current_dir = format!(
+        "{}/greetd.sock",
+        std::fs::canonicalize(&path)
+            .unwrap()
+            .to_string_lossy()
+            .to_string(),
```
Comment (kennylevinsen) [inline fakegreet/src/main.rs]:
```suggestion
            .to_str()
            .unwrap()
```

We don't need an owned string, and `to_string_lossy` may return a string that differs from the input.

Comment (Decodetalkers) [reply fakegreet/src/main.rs]: Ok, I have made the change

**Comment by Decodetalkers** [comment]
By the way, I have make two desktop manager with greetd, one is cli and one is qml, can I add them to your readme? and I like my cli one very much

**Comment by kennylevinsen** [comment]
> By the way, I have make two desktop manager with greetd, one is cli and one is qml, can I add them to your readme?

Sure

**Comment by kennylevinsen** [comment]
Applied, thanks!

**Pull request merged**